release/package.json

```diff
@@ -606,6 +641,7 @@
     "workspaceContains:**/*.fs",
     "workspaceContains:**/*.fsx",
     "workspaceContains:**/*.fsproj",
+    "workspaceContains:**/*.sln",
```

Comment:
This will mean that Ionide is activated for C#-only projects, do we (and users) really want it?